### ISSUE TITLE
origin-manager: workaround lack of notify=False in OBS comment API by not including list of reviews on pending request

### ISSUE THREAD

**Issue by coolo** [openSUSE/openSUSE-release-tools#1897]
https://build.opensuse.org/request/show/683378 was created yesterday 9:39,

9:54 I got the following email from OM:
```
Waiting on reviews of request#683172:

    opensuse-review-team
    openSUSE:Factory:Staging:adi:70
```

Then this morning at 9:40:
```
Waiting on reviews of request#683172:

    openSUSE:Factory:Staging:adi:70
```

Then at 13:50:
```
Previous comment no longer relevant.
```

Now 11 people got these emails - and I can't think of any who need this as notification. And I wouldn't mind if I got one, but I got each of these rounds > 100 times for various KDE packages.

**Comment by jberry-suse** [comment]
Very similar to before and debating not including list of reviews, but figured then it would be requested... The solution has been stated over and over and over...`notify=False` in comment API. So trivial from OBS side it is scary.

Since it will never happen, can reduce this to not include list of reviews (leaper did)...

**Comment by coolo** [comment]
Well, I remember clearly that we discussed to use comments only for actions required. I don't see 'waiting' as a required action

**Comment by jberry-suse** [comment]
Comments used minimally and in this case since the review is not accepted no other forum besides abuse of re-adding review oddity of OBS. It's just interesting that leaper could be even more "spammy" for years and all of the sudden it is declared such. How much better if we could just get actual solutions to problems instead of workarounds.

**Comment by coolo** [comment]
I don't think leaper was more spammy. In my experience it never updated the comment.

**Comment by coolo** [comment]
ok, leaper was spammy on a different level - it commented even fine requests. This didn't feel as spammy though as it always came in timely relation to the original request mail. What changed now is that in the middle of the day I get another hundered emails. Hence it *feels* more spammy.

**Comment by jberry-suse** [comment]
Leaper (and maintenance bot) include same output that includes reviews pending (and leaper would update), but only if line counts changed which means you had a 50/50 chances of looking out outdated information. Just depends on how many requests are submitted in parallel vs already accepted in origin in terms of what you see in emails, but was the same.

https://github.com/openSUSE/openSUSE-release-tools/blob/149e290fae66b68c11bde1b2f4c131da891a231c/check_source_in_factory.py#L127-L136

**Comment by lnussel** [comment]
related to https://github.com/openSUSE/open-build-service/issues/4484
